public/lesswrong-editor/src/ckeditor.js

```diff
@@ -181,7 +181,16 @@ const embedConfig = {
 			`
 		},
 		{
-			name: 'OWID',
+			name: "Manifold",
+			url: /^manifold\.markets\/(?:embed\/)?(\w+\/[\w-]+)$/,
+			html: ([match, longslug]) => `
+				<div data-manifold-id="${longslug}" class="manifold-preview">
```

Comment:
I'm actually not sure what all these data id custom attributes are for!
Unless there's lesswrong-specfic reasons the other embeds have it, the iframe alone should suffice